test/python/test_import_boundaries.py

```
@@ -0,0 +1,97 @@
+
+pytestmark = pytest.mark.L0
+
+_HEAVY = ("torch", "cutlass", "nvidia_cutlass_dsl")
+
+
+def _modules(code: str) -> set:
+    probe = code + "\nimport json, sys; print('@@' + json.dumps(sorted(sys.modules)))"
+    run = subprocess.run([sys.executable, "-c", probe], capture_output=True, text=True)
+    if run.returncode != 0:
+        pytest.skip(f"probe interpreter could not run: {run.stderr.strip()[-200:]}")
+    line = next(l for l in run.stdout.splitlines() if l.startswith("@@"))
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)